test/index.html

```diff
@@ -9,7 +9,33 @@
 <body>
   <div id="qunit"></div>
   <script src="../node_modules/qunitjs/qunit/qunit.js"></script>
-
+  <script type="text/javascript">
```

Comment:
What does this do exactly?

(Also, technically, the `type` isn't necessary in scripts anymore :)